Prepare for 0.14.2 (#1225)
microsoft/vscode-react-native: CHANGELOG.md

## 0.14.2
* Improved extension security [#1219](https://github.com/microsoft/vscode-react-native/pull/1219), [#1222](https://github.com/microsoft/vscode-react-native/pull/1222), [#1223](https://github.com/microsoft/vscode-react-native/pull/1223), [#1224](https://github.com/microsoft/vscode-react-native/pull/1224)
* Updated extension dependencies [#1224](https://github.com/microsoft/vscode-react-native/pull/1224)
* Added support for running iOS debugging on a specific device using flag `target: 'device=<iOS_device_name>'`. [More info](https://github.com/microsoft/vscode-react-native/blob/master/doc/debugging.md#debugging-on-ios-device). [#1207](https://github.com/microsoft/vscode-react-native/pull/1207)

> * Fixed launch of iOS apps where the names of the project and the app bundle are not the same [#1213](https://github.com/microsoft/vscode-react-native/pull/1213)

Reviewer: ```suggestion
* Fixed launch of iOS apps with custom project's configuration: custom scheme name, custom app bundle name. [#1213](https://github.com/microsoft/vscode-react-native/pull/1213)

```
